### ISSUE TITLE
Harmonize all CNV caller modules and automatically convert seg files to projections

### ISSUE THREAD

**Issue by Kdreval** [LCR-BCCRC/lcr-modules#221]
We have so far 4 different CNV callers in lcr-modules: battenberg, sequenza, controlfreec, and cnvkit. They all are very different in the naming convention of the outputs and neither of them does the automatic conversion of generated seg files to the alternative projection.

The liftover module should be automatically imported by each CNV module and perform conversion to alternative genome build and filling of the segments on a per-sample level to make outputs compatible with downstream tools (GISTIC, pyclone, phylowgs etc). The filling of the SEG files should occur in the liftover module to avoid code duplication between modules, and be implemented for both native and lifted SEG files.

The naming convention of the output files should resemble the convention of vcf2maf outputs as close as possible to allow for convenient globbing:
```
CFG["dirs"]["outputs"] + "seg/{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}.{tool}.seg"
```
and
```
CFG["dirs"]["outputs"] + "seg/{seq_type}--projection/{tumour_id}--{normal_id}--{pair_status}.{tool}.{projection}.seg"
```

This can start from generating `grch37` and `hg38` projections. Support for other versions (`hg19` and `grch38`) can be implemented in future CNV merging module or in the later versions of the CNV callers.

**Comment by Kdreval** [comment]
Unfortunately, snakemake does not allow to import of the same snakefile as a sub-module within several module snakefiles as part of a project. It results in conflicting rules/outputs, with whichever module gets loaded latest gets the preference. This update will need a separate script implemented for liftOver, similar to how crossmap is organized within vcf2maf.

**Comment by Kdreval** [comment]
This has been resolved in the PR #224
